recipes/visdcc/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "visdcc" %}
+{% set version = "0.0.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/6f/a3/289b8dec1459807ad6288b41447e9cdad44bff46ea97182a388710f52c7e/{{ name }}-{{ version }}.tar.gz
```

Can you use the predictable URL (see the example) so the bot can monitor for updates

---

@chrisburr  Sorry but this is the only URL I have, would also prefer a predictable one

---

[see the example](https://github.com/conda-forge/staged-recipes/blob/0d7438c23fb07a8075adfb70182f229f9870b8c5/recipes/example/meta.yaml#L14):
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

---

Oh, had no idea, thx for the fix